app/Events/User/UserUpdated.php

```diff
@@ -2,9 +2,38 @@
 
 namespace App\Events\User;
 
+use App\Models\User;
+
 /**
  * User updated event.
  */
 class UserUpdated extends AbstractUserEvent
 {
+    /**
+     * Control of the update log.
+     *
+     * @var bool "send to log" value
+     */
+    protected $sendUpdateToLog;
+
+    /**
+     * Event constructor.
+     *
+     * @param User $user the user
+     */
+    public function __construct(User $user)
+    {
+        parent::__construct($user);
+        $this->sendUpdateToLog = !($user->isDirty('last_access_at') && 2 === count($user->getDirty()));
+    }
+
+    /**
+     * Get the value to check if can send the update to log.
+     *
+     * @return bool
+     */
+    public function sendLog(): bool
```

Comment:
Dato che si tratta di un metodo getter per un valore bool sarebbe opportuno che iniziasse con `is`
```suggestion
    public function isOnlyLastAccessDirty(): bool
```